app/lib/constants/site_config.rb

```diff
@@ -170,15 +170,15 @@ module SiteConfig
         placeholder: SVG_PLACEHOLDER
       },
       logo_png: {
-        description: "Minimum 1024px, used for PWA etc.",
+        description: "Minimum 192x192px, used for PWA etc.",
         placeholder: IMAGE_PLACEHOLDER
       },
       logo_svg: {
         description: "Used as the SVG logo of the community",
         placeholder: SVG_PLACEHOLDER
       },
       main_social_image: {
-        description: "Used as the main image in social networks and OpenGraph",
+        description: "Used as the main image in social networks and OpenGraph. Recommended aspect ratio of 16:9 (600x337px,1200x675px)",
```

Comment:
I've received mixed answers from my google searches. 16:9 is definitely the ideal ratio but the exact pixel around what I listed there.

Reply:
... It's annoying because [Twitter uses `2:1`](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image)....

However, I think we might want to not get too clever with this and just say 16:9 and know that Twitter will clip it a little bit. It should be close enough.